tests/std/tests/P0896R4_counted_iterator_death/test.cpp

```diff
@@ -0,0 +1,248 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#define _CONTAINER_DEBUG_LEVEL 1
+
+#include <cassert>
+#include <concepts>
+#include <iterator>
+#include <vector>
+
+#include <test_death.hpp>
+using namespace std;
+
+int globalArray[5]{10, 20, 30, 40, 50};
+int otherArray[5]{10, 20, 30, 40, 50};
+vector<int> checkedArray{10, 20, 30, 40, 50};
+using vit = vector<int>::iterator;
+
+struct simple_input_iter {
+    using value_type      = int;
+    using difference_type = ptrdiff_t;
+
+    value_type operator*() const {
+        return *_ptr;
+    };

```

Comment:
There are unnecessary semicolons after these member function definitions. Occurs below.

Reply:
Thanks for finding those